pkg/pillar/cmd/vaultmgr/vaultmgr.go

```diff
@@ -351,9 +409,10 @@ func publishVaultStatus(ctx *vaultMgrContext,
 		status.SetErrorNow(fscryptError)
 	} else {
 		args := getStatusParams(vaultPath)
-		if stderr, _, err := execCmd(fscryptPath, args...); err != nil {
+		if stdOut, stdErr, err := execCmd(fscryptPath, args...); err != nil {
+			log.Errorf("Status failed, %v, %v, %v", err, stdOut, stdErr)
 			status.Status = info.DataSecAtRestStatus_DATASEC_AT_REST_ERROR
-			status.SetErrorNow(stderr)
+			status.SetErrorNow(stdOut + stdErr)
```

Comment:
Any idea how large stdout+stderr can be in this case? (Don't want to get 32kbyte of text which nginx might reject on the controller side)

Reply:
it is max 500 characters.

Reply:
OK